ops/terraform/services/pipeline/user-data.sh.tftpl

```diff
@@ -28,7 +35,7 @@ aws ssm get-parameters-by-path \
 cat <<EOF > extra_vars.json
 {
   "data_pipeline_db_url": "${writer_endpoint}",
-  "data_pipeline_jvm_args": "-Xmx{{ ansible_memtotal_mb * 0.80 | int }}m -XX:+PreserveFramePointer",
+  "data_pipeline_jvm_args": "-Xmx{{ ((ansible_memtotal_mb * 0.80) | int) }}m -XX:+PreserveFramePointer",
```

Comment:
Nearly forgot this one, but let's see to pre-allocating the heap. Confirmed with @mackec1 and @brianburton that this isn't just wishful thinking... there are some potential scaling performance gains here to avoid the cost of allocating more heap, etc. What do you think?

```suggestion
  "data_pipeline_jvm_args": "-Xms{{ ((ansible_memtotal_mb * 0.80) | int)m -Xmx{{ ((ansible_memtotal_mb * 0.80) | int) }}m -XX:+PreserveFramePointer",
```

Reply:
Agreed. I can't think of many (or any) downsides and some cursory searching on the topic confirms the benefits. I can make the change.

Reply:
Do we want to consider doing this in `/ops/terraform/modules/resources/templates/fhir_server.tpl` as well?

Reply:
> Do we want to consider doing this in `/ops/terraform/modules/resources/templates/fhir_server.tpl` as well?

I think we ought to consider adopting that, yes! We'll just want to do that in a separate change set.

Reply:
Let's tackle this in [BFD-2391](https://jira.cms.gov/browse/BFD-2391)!